### DISCUSSION TITLE
Ansys 25.1

### DISCUSSION THREAD

**Discussion by OmarZaki96** [ansys/pyfluent#3295] [General]
Hello,

Some of the features I would like to use are only compatible with Ansys 25.1; however, I don't think we have access to Ansys 25.1 yet. Is that correct?

**Comment by ansjschuetz** [comment]
Correct; as the numeric release ID implies, it's scheduled for release in (early) 2025.

**Comment by OmarZaki96** [comment]
Okay, thank you.
